package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/index.js",
   "exports": {
     ".": "./dist/index.js",
-    "./esm": "./esm.mjs"
+    "./esm": "./esm.mjs",
+    "./register": "./register/index.js"
```

Comment:
@cspotcode We’ll also need to include the other `ts-node/register` options, not sure how often they’re used.

Reply:
On it!  Will be pushing out a change in a few minutes.

Reply:
@blakeembrey Thanks, done!